src/collection/AbstractList.php

```diff
@@ -23,7 +23,15 @@
  * @author Cristiano Cinotti
  */
 abstract class AbstractList extends AbstractCollection {
-	use EachPart, IndexFindersPart, ReducePart, RemovePart, ReversePart;
+	use EachPart;
+	use IndexFindersPart {
+		indexOf as parentIndexOf;
+		findLastIndex as parentFindLastIndex;
+		findIndex as parentFindIndex;
```

Comment:
Is there a common understanding of prefixing a mapping to a trait function with `parent*` that I don't know? If I read `$this->parent*` in the code later I would relate it to a method of the super class. If there is such a common methodology I'm fine and I learned it right now, anyway would suggest something like `trait*`?  What you think?

Reply:
Whops, I forgot to find a better name. `trait*` is good for me.

Reply:
fixed